### PR TITLE
Create pyqir metapackage for PyPI

### LINKED ISSUE #88: Create PyQIR metapackage for PyPI
Create pyqir metapackage for PyPI which has the existing pyqir libraries as dependencies.

### PR DESCRIPTION
**Pull request by idavis**
Closes #88

### CODE REVIEW

File: eng/psakefile.ps1 [hunk 1]
```diff
@@ -75,6 +80,16 @@ Task pyqir-tests -Depends init {
     }
 }
 
+Task metawheel {
+    $wheelDir = Split-Path -Parent $wheelhouse
+    if (!(Test-Path $wheelDir)) {
+        New-Item -Path $wheelDir -ItemType Directory | Out-Null
+    }
+    Invoke-LoggedCommand {
+        & $python -m pip wheel --no-deps --wheel-dir $wheelDir $pyqir.meta.dir
```
Comment (cgranade) [inline eng/psakefile.ps1:89]: Ah, is `--wheel-dir` what keeps it from downloading dependencies of the metapackage? Makes sense!

**Review by cgranade — APPROVED** [review]

File: pyqir/setup.cfg [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	Programming Language :: Python :: 3.9
+	Programming Language :: Python :: 3.10
+	Programming Language :: Python
+	Operating System :: Microsoft
+	Operating System :: MacOS
+	Operating System :: Unix
+
+[options]
+package_dir =
+	= .
```
Comment (samarsha) [inline pyqir/setup.cfg:28]: Is this a typo?

Comment (idavis) [reply pyqir/setup.cfg]: @samarsha Thanks! Fixed along with license def in 7e3e280

File: eng/psakefile.ps1 (hunk 1 above)
Comment (idavis) [reply eng/psakefile.ps1:89]: `--no-deps` skips local dep download

**Pull request merged**